packages/core/admin/admin/src/components/Form.tsx

```diff
@@ -333,7 +345,11 @@ const Form = React.forwardRef<HTMLFormElement, FormProps>(
           .filter((el) => el.selected)
           .map((el) => el.value);
       } else {
-        val = value;
+        if (value === '') {
+          val = null;
+        } else {
+          val = value;
+        }
```

Comment:
What case was this a fix for? Maybe we could add a comment explaining it?

Reply:
when you empty a field it needs to be set to null to be considered "cleared" instead of just an empty string. For the API it's an important difference

Reply:
I'll add a note